src/layout/Header.js

```diff
@@ -73,6 +73,7 @@ function Header({ noMargin, noBackground, positionAbsolute }) {
             <NavLink to="/events">Events</NavLink>
             <NavLink to="/pricing">Pricing</NavLink>
             <GuideDropdown />
+            <LogoutButton />
```

Comment:
<img width="180" alt="Screenshot 2019-12-06 at 15 20 01" src="https://user-images.githubusercontent.com/2630/70337968-f0337100-1843-11ea-8240-6ae173d2013e.png">

I think we need a bit of gap between the button and sign in button